spec/gameSpec.js

```diff
@@ -0,0 +1,68 @@
+describe('Game', function() {
+  let game
+
+  beforeEach( () => {
+    game = new Game()
+  })
+  describe('Instantiation', () => {
+        it('should make a new instanct of class', () => {
+          expect(game).toBeInstanceOf(Game)
+        })
+      })
+
+  describe('play', () => {
+      it('calculates the score if the fisrt frame is Strike', () => {
+        frame1 = new Frame(10 , 0)
+        frame2 = new Frame(2 , 5)
+        game.play(frame1)
+        game.play(frame2)
+        expect(game.frames[0].frameScore).toEqual(17)
+      })
+      it('calculates the score if the fisrt frame is Spare', () => {
+        frame1 = new Frame(5 , 5)
+        frame2 = new Frame(2 , 5)
+        game.play(frame1)
+        game.play(frame2)
+        expect(game.frames[0].frameScore).toEqual(12)
+      })
+      it('calculates the score of the first three frames', () => {
+        frame1 = new Frame(10 , 0)
+        frame2 = new Frame(2 , 8)
+        frame3 = new Frame(2 , 4)
+        game.play(frame1)
+        game.play(frame2)
+        game.play(frame3)
+        expect(game.frames[0].frameScore).toEqual(20)
+        expect(game.frames[1].frameScore).toEqual(12)
+        expect(game.score).toEqual(38)
```

Comment:
Great tests! I like how you created new frame instance for testing multiple frames